boards/arm/faze/faze.dts

```diff
@@ -78,7 +78,7 @@
 
 	/* TI LP5030 LED controller connected to I2C1. */
 	led_controller_0: lp5030@30 {
-		compatible = "ti,lp5030", "ti,lp503x";
+		compatible = "ti,lp5030", "ti,lp50xx";
```

Comment:
It was from the original code, I have just replaced the `lp503x` with `lp50xx`. But since this compatible has been removed, let's remove it here also.